### PR TITLE
Feature/spp 826/adjusted value

### PR DESCRIPTION
**Pull request by mattprice-28**
# Description
New adjusted response column added to formview html and python script updated to retrieve new data and add it into the array for display
https://collaborate2.ons.gov.uk/jira/browse/SPP-826

# How to test?
Changes are viewable in the spp826 environment
Have included a sql insert statement in the database repo
https://github.com/ONSdigital/takeon-database/blob/feature/SPP-826/AdjustedValue/story_updates/SPP-826-Adjusted-Response-Value.sql
Only added data for reference 12345678000, script can be used in pgadmin to add responses for other responses if required
New column is observable on all forms, adjusted values will appear as a label (therefore uneditable) where data exists in the database

# Links

### CODE REVIEW

File: app/utilities/combine_data.py [hunk 1]
```diff
@@ -4,14 +4,17 @@ def extract_question_data(form_data, index):
     question_data = {}
     qcode = form_data['view_form_responses'][index]['questioncode']
     response = form_data['view_form_responses'][index]['response']
+    adjusted_response = form_data['view_form_responses'][index]['adjustedresponse']
     question_data['questioncode'] = qcode
     question_data['response'] = response
+    question_data['adjustedresponse'] = adjusted_response
```
Comment (dllmorgan) [inline app/utilities/combine_data.py:10]: Couldn't lines 7 and 10 be replaced with:
`question_data['adjustedresponse'] = form_data['view_form_responses'][index]['adjustedresponse']`

**Review by dllmorgan — DISMISSED** [review]

File: app/utilities/combine_data.py (hunk 1 above)
Comment (mattprice-28) [reply app/utilities/combine_data.py]: Done

**Comment by ranjit-das** [comment]
Tested by modifying the AdjustedResponse value to 'null' in the database. Then searching the same reference, period, survey from UI has failed with - 'Sorry, there is a problem with the service'. Update statement used in database -
update dev01.response
set AdjustedResponse = null
where reference = '12345678000'
and period = '201801'
and Survey = '999A'
and questionCode = '1000'

File: app/utilities/combine_data.py [hunk 2]
```diff
@@ -6,12 +6,14 @@ def extract_question_data(form_data, index):
     response = form_data['view_form_responses'][index]['response']
     question_data['questioncode'] = qcode
     question_data['response'] = response
+    question_data['adjustedresponse'] = form_data['view_form_responses'][index]['adjustedresponse']
```
Comment (sivagudur) [inline app/utilities/combine_data.py:9]: @rhiGriff @mattprice-28  do we need this DateAdjustment parameter in UI form Save submit? I mean as we are not going to Save in Business layer(it is read only field and populated by Validation pipeline) Matt removed this parameter from Save database function during yesterday's review comments. Thanks.

**Review by sivagudur — CHANGES_REQUESTED** [review]

**Review by sivagudur — APPROVED** [review]

**Review by RyanAldy — APPROVED** [review]

**Review by sivagudur — APPROVED** [review]

**Review by RyanAldy — APPROVED** [review]

**Pull request merged**